cli/cmd/volume.go

```diff
@@ -325,3 +327,27 @@ var volumeStatusCmd = &cobra.Command{
 		fmt.Println("STATUS:", volname)
 	},
 }
+
+var volumeExpandCmd = &cobra.Command{
+	Use:   "expand",
+	Short: helpVolumeExpandCmd,
+	Run: func(cmd *cobra.Command, args []string) {
+		validateNArgs(cmd, 2, 0)
+		volname := cmd.Flags().Args()[0]
+		bricks, err := bricksAsUUID(cmd.Flags().Args()[1:])
+		if err != nil {
+			log.WithField("volume", volname).Println("volume expansion failed")
```

Comment:
Same as above, change all messages to "add-brick"

Reply:
Worked on the suggested changes. Thanks.